http/http-advanced/src/test/java/io/quarkus/ts/http/advanced/HttpSseIT.java

```diff
@@ -0,0 +1,32 @@
+package io.quarkus.ts.http.advanced;
+
+import io.quarkus.test.bootstrap.RestService;
+import io.quarkus.test.scenarios.QuarkusScenario;
+import io.quarkus.test.services.QuarkusApplication;
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+@QuarkusScenario
+@Tag("https://github.com/quarkusio/quarkus/issues/36402")
+// this test should only fail on native
```

Comment:
I hope it never fails :-) No, seriously, if you want, you can only do `@EnabledOnNative` or you can just leave comment out. It's fine if you want to keep it.

Reply:
I left this out.